lib/dispatcher/balanced-pool.js

```diff
@@ -146,6 +154,50 @@ class BalancedPool extends PoolBase {
       .map((p) => p[kUrl].origin)
   }
 
+  get [kFree] () {
+    return this[kClients].map(pool => pool[kFree]).reduce((a, b) => a + b, 0)
+  }
+
+  get [kConnected] () {
+    return this[kClients].map(pool => pool[kConnected]).reduce((a, b) => a + b, 0)
+  }
+
+  [kQueueResume] () {
```

Comment:
@metcoder95 You asked elsewhere why I overrode other queue resume code. For one, I couldn't see how other extensions of `PoolBase` or `DispatcherBase` were triggering a queue resume event; if you can point me to where/how that happens that would be fantastic.

But also, this implementation specifically optimizes for finding an available `Pool` with a matching origin.

Thanks for your patience, as this is my first time digging through the Undici codebase, so I'm learning the internals as I go.

Reply:
But is there a specific reason to override it? Or why was required? (maybe I missed something before)